source/features/format-conversation-titles.tsx

```diff
@@ -23,6 +43,10 @@ void features.add({
 		pageDetect.isIssue
 	],
 	init() {
-		observeElement(select('#partial-discussion-header')!.parentElement!, init);
```

Comment:
What if we just change this call to include subtree: true? I don’t like to use it on whole documents, but here the base element is just the header. Would that work?

Reply:
> What if we just change this call to include subtree: true? I don’t like to use it on whole documents, but here the base element is just the header.

`
observeElement(select('#partial-discussion-header')!, init, {});
`

Reply:
> What if we just change this call to include subtree: true? I don’t like to use it on whole documents, but here the base element is just the header.

``` javascript
observeElement(select('#partial-discussion-header')!, init, {
	subtree: true
});
```
> Would that work?

No, because GitHub replaces whole header element on new events (a new message, title, etc).
![Peek 2020-08-19 12-25](https://user-images.githubusercontent.com/36174850/90617418-52d46d80-e217-11ea-9f88-293791ff2998.gif)

Reply:
We can use `onReplacedElement`

``` javascript
		init();
		headerWrapper = select('#partial-discussion-header')!.parentElement!;

		headerWrapper.addEventListener('submit', submitHandler);
	},
	additionalListeners: [
		() => void onReplacedElement('#partial-discussion-header', init)
	],
```

Reply:
> because GitHub replaces whole header element on new events 

That doesn’t sound right, I just pushed a comment and changed the title (in another tab) and the feature kept working after the updates.

Reply:
![](https://user-images.githubusercontent.com/1402241/90628780-1ce2b980-e216-11ea-9299-9ac519593855.gif)

Reply:
I think you mistakenly dropped `parentElement`

> ```js
> observeElement(select('#partial-discussion-header')!, init, {
> 	subtree: true
> });
> ```

The code should be

```diff
- observeElement(select('#partial-discussion-header')!.parentElement!, init);
+ observeElement(select('#partial-discussion-header')!.parentElement!, init, {
+ 	subtree: true
+ });
```

Reply:
> I think you mistakenly dropped `parentElement`

No, it wasn't a mistake. I just thought you were talking
> I don’t like to use it on whole documents

about

``` javascript 
observeElement(select('#partial-discussion-header')!.parentElement!
```

![Screenshot_20200819_142700](https://user-images.githubusercontent.com/36174850/90631536-9d5fe500-e22b-11ea-9e3d-b113880fe1af.png)

Reply:
> The code should be

![Peek 2020-08-19 14-59](https://user-images.githubusercontent.com/36174850/90632541-3fcc9800-e22d-11ea-93a6-73d8464837b0.gif)

Reply:
> The code should be

It works without

`
:not(.rgh-formatted-title)
`

Reply:
with `:not(.rgh-formatted-title)`
![Peek 2020-08-19 15-11](https://user-images.githubusercontent.com/36174850/90633187-47407100-e22e-11ea-9fb3-d3543535540d.gif)
without `:not(.rgh-formatted-title)`
![Peek 2020-08-19 15-10](https://user-images.githubusercontent.com/36174850/90633195-4a3b6180-e22e-11ea-8569-b8506ef59cbf.gif)

Reply:
Indeed, that should be dropped. In this case:

```js
for (const title of select.all('.js-issue-title')) {
	if (!select.exists('a, code', title)) {
		domFormatters.linkifyIssues(title);
		domFormatters.parseBackticks(title);
	}
}